pyproject.toml

```diff
@@ -39,7 +39,23 @@ name = "preliz"
 source = "https://github.com/arviz-devs/preliz"
 tracker = "https://github.com/arviz-devs/preliz/issues"
 
-
+[project.optional-dependencies]
+full = [
+  "arviz",
+  "numpy>=1.22",
+  "scipy>=1.9.1",
+  "matplotlib>=3.5",
+  "nbclient<0.6,>=0.2",
+  "ipywidgets",
+  "ipympl",
+  "jupyterlab",
+  "notebook",
+]
+non-interactive = [
+  "arviz",
+  "numpy>=1.22",
+  "scipy>=1.9.1",
+]
```

Comment:
Assuming we update the dependencies section (see comment above) to:

```
dependencies = [
  "arviz",
  "numpy>=1.22",
  "scipy>=1.9.1",
]
```

I would then update that to one of:

**opinionated version**: use jupyterlab

```
[project.optional-dependencies]
full = [
  "matplotlib>=3.5",
  "nbclient<0.6,>=0.2",
  "ipywidgets",
  "ipympl",
  "jupyterlab",
]
```

**ide agnostic version**:

```
[project.optional-dependencies]
full = [
  "matplotlib>=3.5",
  "nbclient<0.6,>=0.2",
  "ipywidgets",
  "ipympl",
]
lab = ["jupyterlab"]
notebook = ["notebook"]
```

which would modify install commands to:

```shell
pip install preliz # base
pip install "preliz[full,lab]"  # jupyterlab
pip install "preliz[full,notebook]"  # jupyter notebook
```